<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Curver</class>
 <widget class="QWidget" name="Curver">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>531</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>280</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>280</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Растяжение шкалы яркости</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QSlider" name="slider1">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="slider2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="partLinear">
          <property name="text">
           <string>Растянуть</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="rangeLable">
          <property name="text">
           <string>0-255</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Простые кривые</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="1">
       <widget class="QPushButton" name="gammaButton">
        <property name="text">
         <string>Гамма</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QDoubleSpinBox" name="doubleSpinBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QSpinBox" name="spinBox"/>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="solarButton">
        <property name="text">
         <string>Соляризация</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Степенное преобразование со смещенным нулем</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <widget class="QSlider" name="horizontalSlider">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickInterval">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QCheckBox" name="checkBox">
          <property name="text">
           <string>Среднее</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="shiftedButton">
          <property name="text">
           <string>Выполнить</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="sliderLabel">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" alignment="Qt::AlignRight">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>1/N:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="shiftedSpinbox"/>
        </item>
        <item row="3" column="1">
         <widget class="QRadioButton" name="ngo">
          <property name="text">
           <string>n&gt;1</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QRadioButton" name="nlo">
          <property name="text">
           <string>n&lt;1</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
